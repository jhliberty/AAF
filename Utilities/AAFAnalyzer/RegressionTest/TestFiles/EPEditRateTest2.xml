<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE aaf-file SYSTEM "EPDerivationChain.dtd">
<aaf-file>
<!-- Test one incorrect audio edit rate -->
 <clip name="Clip 1">
  <timeline-mob-slot edit-rate-denominator="24" edit-rate-numerator="600">
   <source-clip length="0" track-type="audio">
    <file-source name="File Source 1" sample-rate-denominator="24" sample-rate-numerator="600">
     <timeline-mob-slot edit-rate-denominator="1" edit-rate-numerator="1">
      <eoc length="0" track-type="none"/>
     </timeline-mob-slot>
    </file-source>
   </source-clip>
  </timeline-mob-slot>
 </clip>
 <clip name="Clip 2">
  <timeline-mob-slot edit-rate-denominator="24" edit-rate-numerator="1200000">
   <source-clip length="0" track-type="audio">
    <file-source name="File Source 1" sample-rate-denominator="24" sample-rate-numerator="600">
     <timeline-mob-slot edit-rate-denominator="1" edit-rate-numerator="1">
      <eoc length="0" track-type="none"/>
     </timeline-mob-slot>
    </file-source>
   </source-clip>
  </timeline-mob-slot>
 </clip>
</aaf-file>
