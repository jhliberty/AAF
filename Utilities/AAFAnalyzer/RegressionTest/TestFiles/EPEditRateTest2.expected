        Name:   Edit Protocol Edit Rate Test
        Desc:   Verify that all essence tracks have valid edit rates.
        Result: Failed
        Reason: 
        Covered Requirements: REQ_EP_091; REQ_EP_092; REQ_EP_100
        Noted Requirements: None
        Passing Requirements: None
        Info Requirements: None
        Warning Requirements: None
        Failing Requirements: REQ_EP_099

            Name:   Audio Track Rates
            Desc:   All audio tracks within a file shall have the same edit rate and sample rate.
            Result: Failed
            Reason: All audio tracks within the file do not have the same edit rate.
            Covered Requirements: None
            Noted Requirements: None
            Passing Requirements: None
            Info Requirements: None
            Warning Requirements: None
            Failing Requirements: REQ_EP_099
            Detail: 1 audio track(s) have an edit rate of 600/24.
            Detail: 1 audio track(s) have an edit rate of 1200000/24.

